packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -14,6 +13,7 @@ import {
   initializeComponentRef
 } from '../../Utilities';
 import { IList, IListProps, IPage, IPageProps, ScrollToMode } from './List.types';
+import { EventListener } from '@fluentui/react-component-event-listener';
```

Comment:
Given that `EventGroup` is still in many other components, all this will do is increase the bundle size for all list consumers. Is there any way to somehow use a dependency injection scheme to have the host plug this in?

Reply:
If we did that we'd be stuck with a dependency injection pattern semi-permanently for what should be a temporary problem (we're working on removing eventgroup and basecomponent in all our code).

Reply:
Can we perform the `EventGroup` removal refactor in a single change-set to minimize the bundle size regression?

Once merged, we'd still have both `EventGroup` and `@fluentui/react-component-event-listener` in ODSP since many of our internal components use `EventGroup`, but the diff should be less.